src/PostalRegistry.Api.Legacy/appsettings.json

```diff
@@ -1,19 +1,26 @@
 {
   "ConnectionStrings": {
-    "Events": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.PostalRegistry;Trusted_Connection=True;",
-    "LegacyProjections": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.PostalRegistry;Trusted_Connection=True;",
-    "LegacyProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.PostalRegistry;Trusted_Connection=True;",
-    "SyndicationProjections": "Server=(localdb)\\mssqllocaldb;Database=postalregistry-dev;Trusted_Connection=True;",
-    "SyndicationProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=postalregistry-dev;Trusted_Connection=True;"
+    "Events": "Server=aocsrv399;Database=ldes;Trusted_Connection=True;",
```

Comment:
don't check in. Use your own appsettings config file for local connection strings.
`appsettings.PCHOSTNAME.json`